plugins/pow/parameters.go

```diff
@@ -9,7 +9,7 @@ import (
 // ParametersDefinition contains the definition of the parameters used by the pow plugin.
 type ParametersDefinition struct {
 	// Difficulty defines the PoW difficulty.
-	Difficulty int `default:"22" usage:"PoW difficulty"`
```

Comment:
This should also be changed in `config.default.json` and possibly across documentation

Reply:
Done, I couldn't find references to this number in the docs.